modules/oadp-using-data-mover-for-csi-snapshots.adoc

```diff
@@ -32,7 +37,7 @@ Currently, Data Mover does not support Google Cloud Storage (GCS) buckets.
 +
 [NOTE]
 ====
-The VolSync Operator is required only for use with the Technology Preview Data Mover. The Operator is not required for using OADP production features.
+The VolSync Operator is not required for using OADP production features.
```

Comment:
Is this note relevant for the current version that has OADP production features?

Reply:
Data mover is planned to be a supported feature in OADP 1.2 so as of 1.2, volsync operator may be a pre-requisite if user intend to use data mover functionality.

Reply:
It is only required for the data mover feature in OADP

Reply:
"The VolSync Operator is not required for using OADP production features." should be changed to..  "The VolSync Operator is required for using OADP datamover feature."